gpameter.user.js

```diff
@@ -27,9 +27,15 @@ var grade_values = {
 };
 console.log(studentId);
 add_checkboxes = function(){
+	var mapElems = new Map();
```

Comment:
Since we are just using this to check presence and the associated value is irrelevant, how about using a [Set][1] instead?

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set